proposed/extended-coding-style-guide-meta.md

```diff
@@ -23,13 +23,13 @@ but ultimately this meant it was a compromise of many of the different projects'
 The repercussions of projects changing their coding guidelines to align with PSR-2 (Almost
 all projects do align with PSR-1, even if it is not explicitly stated) were seen to be too
 great (losing git history, huge changesets and breaking existing patches/pull requests) and
-this therefore hurt adoption of PSR-2. Therefore, whilst PSR-N will be more prescriptive
+this therefore hurt adoption of PSR-2. Therefore, whilst PSR-12 will be more prescriptive
```

Comment:
> ... and this therefore hurt adoption of PSR-2. Therefore, whilst PSR-12 will be more prescriptive than descriptive of the projects, but this is in the hope that it will ..."

This hurts my brain.. I think what this is trying to say is:

> PSR-2 required adopters to reformat large amounts of existing code which stifled adoption. To help alleviate this issue with PSR-12, we have taken a more prescriptive approach and defined the standards for new language features as they are released. We hope that because this specification is defined prior to mass amounts of code being written, it will have a better chance of being adopted.

Thoughts?